packages/manager/modules/vps/src/dashboard/tile/configuration/upgrade/translations/Messages_fr_FR.json

```
@@ -0,0 +1,11 @@
+{
+  "vps_dashbaord_tile_configuration_upgrade_memory_action_title": "Améliorer la mémoire de votre VPS de 2 à 4 Go",
+  "vps_dashbaord_tile_configuration_upgrade_memory_action_details": "Vous êtes sur le point d‘ajouter 2 Go de mémoire à votre VPS.",
+  "vps_dashbaord_tile_configuration_upgrade_storage_action_title": "Améliorer le stockage de votre VPS de 40 à 80 Go",
+  "vps_dashbaord_tile_configuration_upgrade_storage_action_details": "Vous êtes sur le point d‘ajouter 40 Go de stockage à votre VPS.",
```

```suggestion
  "vps_dashbaord_tile_configuration_upgrade_storage_action_details": "Vous êtes sur le point d‘ajouter {{quantity}} de stockage à votre VPS.",
```
To be future proof ?